### DISCUSSION TITLE
Unable to import existing osu!lazer data folder on first-run setup/fresh install

### DISCUSSION THREAD

**Discussion by mty22** [ppy/osu#19781] [Q&A]
I was unable to select my existing osu!lazer install on a separate drive during the first run setup guide as the actual 'Select' button appears to be missing.

I was however, able to import everything via 'Change folder location...' within the settings.

OS: Windows 11 (Ver: 21H2, Build: 22000.856) -- Note: fresh install from scratch.
osu!lazer build: 2022.810.2
Video: https://www.youtube.com/watch?v=tz6jbyBdd98

Unfortunately I do not have the logs as I re-opened my game client after changing the folder location.

Thanks!

frzn (osu!) / frznmatt (Twitch).

**Comment by peppy** [comment]
Hmm, the first run overlay doesn't actually have any support for selecting a lazer install, only stable.

Maybe a bit of an oversight.

**Reply by ShayBox** [reply]
should change it to say osu!stable on that screen, it's confusing

**Comment by Geminite** [comment]
Is it supported now?

**Reply by peppy** [reply]
You can use the "change folder location" method as mentioned in the opening post.
